torch/csrc/jit/tensorexpr/codegen.h

```diff
@@ -66,6 +66,17 @@ class TORCH_API CodeGen {
 
   virtual void call(const std::vector<CallArg>& args) = 0;
 
+  virtual at::Tensor empty_strided(
+      c10::IntArrayRef size,
+      c10::IntArrayRef stride,
+      c10::optional<c10::ScalarType> dtype_opt,
+      c10::optional<c10::Layout> layout_opt,
+      c10::optional<c10::Device> device_opt,
+      c10::optional<bool> pin_memory_opt) {
+    return at::empty_strided(
```

Comment:
do we ever use the `at::empty_strided` overload ? is it necessary ?

Reply:
the SimpleIREval interpreter will use it.  I guess since the interpreter's running on cpu I could just make it call at::native::empty_strided_cpu